lib/spack/spack/build_systems/python.py

```diff
@@ -127,7 +127,9 @@ def import_modules(self):
             list: list of strings of module names
         """
         modules = []
-        root = self.spec['python'].package.get_python_lib(prefix=self.prefix)
+        root = os.path.join(
+            self.prefix, self.spec['python'].package.config_vars['false']['false']
```

Comment:
should this have some other index for `config_vars` or is it really `['false']['false']`?

Reply:
It's correct. When you serialize and deserialize JSON it converts bools to strings.